### PR TITLE
Implemented getAllSkirts with URL api/products/skirts.

### PR DESCRIPTION
**Pull request by parthiban-samykutti**
Created a Controller, Repository and test for api/products/skirts; Implemented getAllSkirts.

### CODE REVIEW

**Comment by mosdevly** [comment]
There are 3 tests in this PR. There should only be 1.

**Comment by DurazoJoe** [comment]
@Protosac We've removed the two additional tests that had been created due to starting with the nzombie approach

**Review by shashi2k6 — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -17,7 +17,10 @@ dependencies {
     runtimeOnly 'mysql:mysql-connector-java'
 	runtimeOnly 'com.h2database:h2'
     testImplementation('org.springframework.boot:spring-boot-starter-test')
-
+    compileOnly 'org.projectlombok:lombok:1.18.16'
```
Comment (mosdevly) [inline build.gradle]: Theres been updates to master that added Lombok to the project. You might want to pull in those updates and resolve the conflict.

Comment (DurazoJoe) [reply build.gradle]: Done!

Comment (DurazoJoe) [reply build.gradle]: Resolved the conflict

Comment (mosdevly) [reply build.gradle]: Still see the build failing here. It shows that one your tests is failing.

File: src/main/java/com/cottonfactory/products/controllers/SkirtsController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/products/")
+public class SkirtsController {
```
Comment (mosdevly) [inline src/main/java/com/cottonfactory/products/controllers/SkirtsController.java:17]: 1 teeny tiny update needed: add the `@Api` annotation to the controller. See the README for details!

**Review by mosdevly — CHANGES_REQUESTED** [review]
1 teeny tiny update needed: add the `@Api` annotation to the controller. See the README for details!

File: src/main/java/com/cottonfactory/products/controllers/SkirtsController.java (hunk 2 above)
Comment (parthiban-samykutti) [reply src/main/java/com/cottonfactory/products/controllers/SkirtsController.java:17]: done

File: build.gradle (hunk 1 above)
Comment (parthiban-samykutti) [reply build.gradle]: resolved and build is successful.

**Review by RichFromGalvanize — APPROVED** [review]
Looks good! Good job!

**Comment by DurazoJoe** [comment]
> Looks good! Good job!

Thanks, it still shows the change request from Kay as active but we solved it and marked it as resolved, hence the merging is blocked, what can we do about this?

**Review by mosdevly — APPROVED** [review]

**Pull request merged**